packages/perseus/src/widgets/numeric-input/numeric-input.tsx

```diff
@@ -345,10 +345,10 @@ const propsTransform = function (
 const propUpgrades = {
     /* c8 ignore next */
     "1": (initialProps: any): PerseusNumericInputWidgetOptions => {
-        // If the initialProps has a value, it means we're upgrading from
+        // If the initialProps has simplify, it means we're upgrading from
         // input-number to numeric-input. In this case, we need to upgrade
         // the widget options accordingly.
-        if (initialProps.value) {
+        if (initialProps.simplify) {
```

Comment:
Given we were bitten by this type of "bare if()" check for `value`, I think it's much safer and clearer to use an explicit check against `undefined`.  It appears that `simplify` is never null or an empty string in our published content (at least `"en"`), but I think we should move away from these truthy/falsey type of checks given how dangerous they can be in the face of `0` and `""`.

Reply:
BTW, I used this shell command against the downloaded content tree to check: 

```
$ rg '"type"\s*:\s*"input-number"' -l | xargs jq --slurp '.. | objects | select(has("widgets")) | .widgets[].options | select(has("value") and .value == 0) | .simplify' | sort | uniq
```